### DISCUSSION TITLE
Table sorting

### DISCUSSION THREAD

**Discussion by ac5tin** [TaKO8Ki/gobang#122] [General]
Is there a way to sort a table by a specific column? Maybe say i can use the arrow keys to go to the target column and then press `s` to sort?

**Comment by ac5tin** [comment]
i think i can sort by modifying the filter, i have to manually put `order by created_at desc`  to sort it currently

**Reply by TaKO8Ki** [reply]
Currently you need to do it to sort a table by a specific filter. I'm going to implement `s` key to sort a table by selected column key.

**Reply by TaKO8Ki** [reply]
I will open issue about it.
